form/form.go

```diff
@@ -35,12 +38,63 @@ type Filler interface {
 	Fill(f Form) (map[string]interface{}, error)
 }
 
-// PromptingFiller fills out a set of environschema fields by prompting
-// the user for each field in sequence.
-type PromptingFiller struct {
-	// The Prompter to use to get the responses. If this is nil then
-	// DefaultPrompter will be used.
-	Prompter Prompter
+// SortedFields returns the given fields sorted first by group name.
```

Comment:
Not for this PR but I think we should probably use names rather than descriptions everywhere.